terraform/modules/data_share_service/eventbridge.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_cloudwatch_event_rule" "tunnel_notification" {
+  name        = "bastion-tunnel-notification"
+  description = "Send an SNS notification when a tunnel is initiated to the bastion host"
+
+  event_pattern = jsonencode({
```

Comment:
So this is just starting a session, do we want to have anything about where the tunnel is going?

Reply:
Yeah I originally wasn't sure how this is done, but I've had a further dig into the docs & cloudtrail logs and think I might have found a way to do it